Purchases/Public/RCPurchases.m

```diff
@@ -290,7 +290,12 @@ - (instancetype)initWithAppUserID:(nullable NSString *)appUserID
         };
 
         [self.identityManager configureWithAppUserID:appUserID];
-        [self updateAllCachesWithCompletionBlock:callDelegate];
+        if (!self.systemInfo.isApplicationBackgrounded) {
+            [self updateAllCachesWithCompletionBlock:callDelegate];
+        } else {
+            [self sendCachedPurchaserInfoIfAvailable];
```

Comment:
this bit is so that if there's something cached we still call the delegate. I believe this is a departure from Android. @vegaro was there a specific reason not to call the delegate there? I'm worried that not calling it is a bit inconsistent with the philosophy of sending whatever is cached as fast as we can.

Reply:
You're right, we should be calling in Android. I think that's what I would expect as a developer.